exercises/01_penguin_classification.ipynb

```diff
@@ -434,51 +487,154 @@
     "Have a go at importing the model weights for a large model like ResNet50"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import ResNet50 here\n",
+    "from torchvision.models import resnet50\n",
+    "\n",
+    "# Create a ResNet50 model and print it.\n",
+    "model = resnet50(pretrained=True)\n",
+    "# Set the model to evaluation mode.\n",
+    "model.eval()\n",
+    "# Print the model architecture.\n",
+    "print(model)"
+   ]
+  },
```

Comment:
In the original workshop we taught participants how to create a net from scratch and introduced loading pre-trained nets in a later exercise.

I see creating a net is still covered in task 6, so am not sure what this part is for?

Reply:
Hmm, yes, I see the argument about overwriting a model with something else. This definitely needs to change. I will put the example in markdown and just have it there in true blue peter style (Here's one that was trained earlier). I'll change the variable `model` to something else so it doesn't corrupt the namespace.

Reply:
My justification for including it was just to show this is how you can get a complicated model online and interrogate it's structure., to draw parallels with the simple one we've defined.